### ISSUE TITLE
foss2022a build on ubuntu 22.04

### ISSUE THREAD

**Issue by elc42** [easybuilders/easybuild#822]
Hello,
Buildinf foss-2022a.eb on ubuntu 22.04 leads to the following error:

```
== processing EasyBuild easyconfig /home/elcar/.local/easybuild/easyconfigs/l/libreadline/libreadline-8.1.2-GCCcore-11.3.0.eb
== building and installing libreadline/8.1.2-GCCcore-11.3.0...
== fetching files...
== creating build dir, resetting environment...
== unpacking...
== patching...
== preparing...
== configuring...
== FAILED: Installation ended unsuccessfully (build directory: /home/elcar/.local/easybuild/build/libreadline/8.1.2/GCCcore-11.3.0): build failed (first 300 chars): cmd " ./configure --prefix=/home/elcar/.local/easybuild/software/libreadline/8.1.2-GCCcore-11.3.0  --build=/bin/bash: /home/elcar/.local/easybuild/software/ncurses/6.3-GCCcore-11.3.0/lib/libtinfo.so.6: no version information available (required by /bin/bash)
x86_64-pc-linux-gnu  --host=/bin/bash: /h (took 1 secs)
== Results of the build can be found in the log file(s) /tmp/eb-igmtani_/easybuild-libreadline-8.1.2-20220914.211003.PcQvl.log
ERROR: Build of /home/elcar/.local/easybuild/easyconfigs/l/libreadline/libreadline-8.1.2-GCCcore-11.3.0.eb failed (err: 'build failed (first 300 chars): cmd " ./configure --prefix=/home/elcar/.local/easybuild/software/libreadline/8.1.2-GCCcore-11.3.0  --build=/bin/bash: /home/elcar/.local/easybuild/software/ncurses/6.3-GCCcore-11.3.0/lib/libtinfo.so.6: no version information available (required by /bin/bash)\nx86_64-pc-linux-gnu  --host=/bin/bash: /h')
```

**Comment by elc42** [comment]
https://github.com/easybuilders/easybuild-easyconfigs/commit/d1adb4ae5a7af17b69a17dd75dd729fe1c56bb06 to ncurses-6.3-GCCcore-11.3.0.eb solves the problem

**Comment by boegel** [comment]
This problem was indeed fixed in #16270, those changes are included with EasyBuild v4.6.2
